### DISCUSSION TITLE
Contribuição com a comunidade e Hacktoberfest

### DISCUSSION THREAD

**Discussion by joaomota59** [joaomota59/diversos#12] [Ideas] [answered]
Todo ano há um evento [hacktoberfest](https://www.zup.com.br/blog/hacktoberfest) que basicamente têm como finalidade ajudar a comunidade Open Source, o que acham de tornar esse repositório, viável( para esse evento), para as pessoas contribuírem com códigos para a comunidade do git?

**Comment by filipeas** [comment, marked as answer]
segundo esse site aqui https://www.zup.com.br/blog/hacktoberfest teria que ser um repositório com um projeto open source específico, porém nesse repositório tem códigos muito variados. Precisaria especificar o objetivo desse repositório pra poder tentar entrar num envento desses não?

**Reply by joaomota59** [reply]
Entendi, farei isso mesmo!
